src/en/users/survey/index.html

```diff
@@ -0,0 +1,34 @@
+---
+title: Survey
+order: 3
+---
+
+<section class="section">
+  <div class="grid lg:grid--cols-2 grid--gap-8 lg:grid--gap-28">
+    <div>
+      <h2 class="h2">Make your voice heard!</h2>
+
+      <p class="p">
+        This information will help guide the Ceph community’s investment in the future development of Ceph and the Ceph community.
+      </p>
+
+      <p class="p">
+        The <a class="a" href="https://ceph.io/en/community/meetups/">discussions</a> don’t stop with a strong
+        <a class="a" href="https://ceph.io/en/community/">Community</a>, a team of
+        <a class="a" href="https://ceph.io/en/community/team/">Daring Leaders</a>, and our
+        <a class="a" href="https://ceph.io/en/foundation/members/">Supporters</a>.
+      </p>
+
+      <a class="button" href="https://survey.zohopublic.com/zs/tLCskv">Take the survey</a>
+    </div>
+    <div>
+      <h2 class="h2">Past Results</h2>
+
+      <ul>
+        <li><a class="a" href="https://ceph.io/en/news/blog/2018/ceph-user-survey-2018-results/">2018</a></li>
+        <li><a class="a" href="https:///ceph.io/en/news/blog/2020/ceph-user-survey-2019/">2019</a></li>
```

Comment:
This doesn't seem to have the results, is this expected?

Reply:
Yeah the download link to a pdf gives 404.

Reply:
I'm not sure why the ods link isn't working. I can open the file from the ceph.io repo, but going to the blog post and clicking the link gives a 404 as reported.

https://ceph.io/en/news/blog/2018/ceph-user-survey-2018-results/
https://ceph.io/en/news/blog/2018/ceph-user-survey-2018-results/attachments/Ceph-User-Survey-2018-For-Distribution.ods

@djgalloway, do you know of anything that would cause this?

Reply:
`https:///` ?

Also, try adding the ods file extension to .eleventy.js like Adam did in 4b6bd423dbb95616fe7baa0c93022767f49b9f17